modules/ossm-member-roll-create.adoc

```diff
@@ -102,3 +101,41 @@ spec:
 ----
 +
 . Modify the default YAML to add your projects as `members`.  You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+
+[id="ossm-member-roll-create-member_{context}"]
+== Creating the {ProductName} members
+
+`ServiceMeshMember` resources are created by service mesh users who are not administrators. While project administrators are automatically given permission to create the `ServiceMeshMember` resource in their project, they are not allowed to point it to any `ServiceMeshControlPlane` until the service mesh administrator explicitly grants access to the service mesh. Administrators can modify the mesh users role binding in the control plane project to specify the users and groups that are granted access.
+
+The `ServiceMeshMember` adds the project to the `ServiceMeshMemberRoll` within the control plane project it references. 
+
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMember
+metadata:
+  name: default
+spec:
+  controlPlaneRef:
+    namespace: control-plane-namespace
+    name: minimal-install
+----
+
+The mesh users role binding is created after the administrator creates the `ServiceMeshControlPlane` resource. The administrator can also create it beforehand. For example, the administrator can create it in the same `oc apply` operation as the `ServiceMeshControlPlane` resource.
```

Comment:
Maybe add a sentence saying that you need to edit the existing role binding (and how to do it).

Reply:
I think we should tell people to use `oc policy add-role-to-user`, which is a simpler approach.  Not sure if this is easy to do through the console though.